docs/en/content/writing-markdown-latex/index.md

```diff
@@ -244,6 +244,24 @@ To **dynamically theme images** according to the user's light or dark theme
 
 Alternatively, paste a **themeable inline SVG image** in the form `<svg fill="currentColor">...</svg>` directly into your Markdown.
 
+**Full list of figure options:**
```

Comment:
Please add the documented option Values/Description fields from https://github.com/wowchemy/wowchemy-hugo-themes/blob/main/modules/wowchemy/layouts/shortcodes/figure.html so we can document all the figure options in the docs site and not just in the source